docker/requirements.txt

```diff
@@ -0,0 +1,7 @@
+fastapi
```

Comment:
Should version all these dependencies

Reply:
I just pinned them to what I was building with locally